engine/src/test/java/org/camunda/bpm/engine/test/api/optimize/GetHistoricOperationLogsForOptimizeTest.java

```diff
@@ -98,36 +106,395 @@ public void cleanUp() {
     for (User user : identityService.createUserQuery().list()) {
       identityService.deleteUser(user.getId());
     }
-    for (Group group : identityService.createGroupQuery().list()) {
-      identityService.deleteGroup(group.getId());
-    }
-    for (Authorization authorization : authorizationService.createAuthorizationQuery().list()) {
-      authorizationService.deleteAuthorization(authorization.getId());
-    }
     ClockUtil.reset();
     identityService.clearAuthentication();
+    helper.removeAllRunningAndHistoricBatches();
   }
 
   @Test
-  public void getHistoricUserOperationLogs() {
-     // given
+  public void getHistoricUserOperationLogs_suspendProcessInstanceByProcessInstanceId() {
+    // given
+    BpmnModelInstance simpleDefinition = Bpmn.createExecutableProcess("process")
+      .startEvent("startEvent")
+      .userTask("userTask")
+      .name("task")
+      .endEvent("endEvent")
+      .done();
+    testHelper.deploy(simpleDefinition);
```

Comment:
This code is repeated in all tests in more or less the same way. I propose to move it to an `@Before` method.

Reply:
Good idea!